### ISSUE TITLE
Sorting output

### ISSUE THREAD

**Issue by JonnyLomond** [JonnyLomond/FlailingCanary#1]
We are have a method which iterates through lists of dictionaries, and dictionaries whose values are themselves dictionaries. How do we sort a list of dictionaries?

It is not a big issue because the ordering shouldn't matter when writing to the database. When we have decided what fields to use, perhaps we should make another print to screen listener which will print our specified fields.

**Comment by AdamSturge** [comment]
I agree. Perhaps a utilities.py script to contain some common methods.
